app/api/chat/welcome/route.ts

```
@@ -0,0 +1,108 @@
+        global.broadcastEvent("chat.created", chatEventData, [sessionId]);
+      }
+
+      return Response.json(chat);
+    } catch (uniqueError) {
+      // If "welcome" ID is already taken, fallback to a session-specific ID
+      if (
+        uniqueError instanceof Error &&
+        uniqueError.message.includes("Unique constraint")
+      ) {
+        console.log("Welcome chat ID collision, using session-specific ID");
+        const fallbackId = "welcome-" + sessionId.substring(0, 8);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)